mlflow/utils/request_utils.py

```diff
@@ -24,6 +24,7 @@
         504,  # Gateway Timeout
     ]
 )
+DOWNLOAD_CHUNK_TIMEOUT_100M = 10 * 60
```

Comment:
What does `100M` mean?

Reply:
It means for 100MB files we set timeout to 10minutes, so the timeout for different size files are scaled according to this number

Reply:
/ai suggest names for this constant

Reply:
/ai Let me give you more context. This value is used like this to compute a timeout value. `range_end - range_start` represents the size of a chunk.

```python
timeout = DOWNLOAD_CHUNK_TIMEOUT_100M * (range_end - range_start) / 100_000_000
```